app/client/cli/actor.go

```diff
@@ -0,0 +1,234 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/big"
+	"regexp"
+	"strings"
+
+	sharedTypes "github.com/pokt-network/pocket/shared/types"
+	"github.com/pokt-network/pocket/utility/types"
+	utilityTypes "github.com/pokt-network/pocket/utility/types"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	subcmdFlags := []cmdOption{func(c *cobra.Command) {
+		c.Flags().StringVar(&pwd, "pwd", "", "passphrase used by the cmd, non empty usage bypass interactive prompt")
+	}}
+	rootCmd.AddCommand(NewActorCommands(subcmdFlags)...)
+}
+
+var (
+	// DISCUSS(team): this should probably come from somewhere else
+	stakingRecommendationAmount = big.NewInt(15100000000)
+	pwd                         string
+)
+
+type (
+	cmdOption   func(*cobra.Command)
+	actorCmdDef struct {
+		Name      string
+		ActorType types.ActorType
+		Options   []cmdOption
+	}
+)
+
+func NewActorCommands(cmdOptions []cmdOption) (cmds []*cobra.Command) {
+	actorCmdDefs := []actorCmdDef{
+		{"Application", types.ActorType_App, cmdOptions},
+		{"Node", types.ActorType_Node, cmdOptions},
+		{"Fisherman", types.ActorType_Fish, cmdOptions},
+		{"Validator", types.ActorType_Val, cmdOptions},
+	}
+
+	for _, cmdDef := range actorCmdDefs {
+
+		cmd := &cobra.Command{
+			Use:     cmdDef.Name,
+			Short:   fmt.Sprintf("%s actor specific commands", cmdDef.Name),
+			Aliases: []string{strings.ToLower(cmdDef.Name), cmdDef.ActorType.GetActorName()},
+			Args:    cobra.ExactArgs(0),
+		}
+		cmd.AddCommand(newActorCommands(cmdDef)...)
+		cmds = append(cmds, cmd)
+	}
+
+	return cmds
+}
+
+func newActorCommands(cmdDef actorCmdDef) (cmds []*cobra.Command) {
+	stakeCmd := &cobra.Command{
+		Use:   "Stake <fromAddr> <amount> <RelayChainIDs> <serviceURI>",
+		Short: "Stake a node in the network. Custodial stake uses the same address as operator/output for rewards/return of staked funds.",
+		Long: `Stake the node into the network, making it available for service.
+Will prompt the user for the <fromAddr> account passphrase. If the node is already staked, this transaction acts as an *update* transaction.
+A node can updated relayChainIDs, serviceURI, and raise the stake amount with this transaction.
+If the node is currently staked at X and you submit an update with new stake Y. Only Y-X will be subtracted from an account
+If no changes are desired for the parameter, just enter the current param value just as before`,
+		Args: cobra.ExactArgs(4),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(pocket/issues/150): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+			// currently ignored since we are using the address from the PrivateKey
+			// fromAddr := crypto.AddressFromString(args[0])
+			amount := args[1]
+
+			am, err := sharedTypes.StringToBigInt(args[1])
+			if err != nil {
+				log.Fatal(err)
```

Comment:
Right, I am now returning since main.go takes care of doing a `log.Fatal` if `err != nil`

Addressed in https://github.com/pokt-network/pocket/pull/177/commits/5e185e47f1fa61fe479a687d9058492d8870c794